### DISCUSSION TITLE
Is it possible to freeze column headings when scrolling down on PnP search results WebPart?

### DISCUSSION THREAD

**Discussion by itebet** [microsoft-search/pnp-modern-search#3575] [Q&A] [answered]
It's requested from my business users due we returned a long list of results (internal directives) and compliance team wants to keep the headers on top when users scroll down the list.
Of course, list can be shorter adding filters, but they want to bring this option by default without any filter.

**Comment by kasperbolarsen** [comment]
No, not at the moment, but we would welcome any PR from you that would provide this functionality :-)

**Reply by wobba** [reply]
You can probably achieve this using CSS in a custom template.

**Comment by wobba** [comment, marked as answer]
Fixed in in #3580 for the next release :) Thank you @patrikhellgren.

**Reply by itebet** [reply]
That's completely awesome, thanks a lot! 
:-)
